src/renate/benchmark/experiment_config.py

```diff
@@ -331,7 +333,9 @@ def test_transform(dataset_name: str) -> Optional[Callable]:
     if dataset_name in ["CLEAR10", "CLEAR100"]:
         return transforms.Compose(
             [
-                transforms.Resize(224),
+                transforms.Resize(
```

Comment:
is this following the pre-processing described in the CLEAR paper?

Reply:
In the paper, they work on PIL images. Then this is not required. We work on Tensors. According to the documentation, this should bring it closer to what you would do when working on PIL images. Check the part on `antialiasing`: https://pytorch.org/vision/main/generated/torchvision.transforms.Resize.html